lib/internal/Magento/Framework/App/Action/HttpHeadActionInterface.php

```diff
@@ -1,21 +0,0 @@
-<?php
-/**
```

Comment:
@markshust Can we please have this file back?

Reply:
@manavluhar @nuzil the whole point of this ticket was to remove this file, as it is deprecated, only used in one spot, and shouldn't be causing failing tests. I'm unsure why someone would want to keep this file around.

Reply:
@markshust backward compatibility, @nuzil can add further.

Reply:
I thought that's what SemVer and the patch releases are for. How long will cruft like this build up if we never have plans to remove it?

Reply:
Got your point @markshust, let the maintainer review it.